README.md

```diff
@@ -148,27 +151,27 @@ If no `conventionalCommits` are defined in the `settings-file`, the commit types
         "release": "none"
     },
     "style": {
-        "release": "none"
+        "release": "patch"
     },
     "refactor": {
-        "release": "none"
+        "release": "minor"
     },
     "perf": {
-        "release": "none"
+        "release": "minor"
     },
     "test": {
-        "release": "none"
+        "release": "patch"
     }
 }
 ```
 
 examples:
 
 - break: LANZ-123 updated API design-> creates a major release (**X+1**.y.z)
-- fix!: LANZ-123 fixed bug but breakign compatibility-> creates a major release (**X+1**.y.z)
+- fix!: LANZ-123 fixed bug but breaking compatibility-> creates a major release (**X+1**.y.z)
```

Comment:
I don't think our action supports the "!" exclamation feature. I think this example is wrong.